modules/aws-bastion-instance/main.tf

```diff
@@ -77,13 +72,13 @@ resource "aws_iam_instance_profile" "bastion" {
   role        = aws_iam_role.bastion.id
 }
 
-resource "aws_launch_configuration" "bastion" {
-  name_prefix                 = "${var.name}-"
-  image_id                    = var.ami_id
-  instance_type               = var.instance_type
-  user_data                   = data.template_file.user_data.rendered
-  enable_monitoring           = var.enable_monitoring
```

Comment:
Bring back an option to enable detailed monitoring - rename variable to `detailed_monitoring` and use `monitoring` on `aws_instance` resource.